components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/IdentityManagementEndpointUtil.java

```diff
@@ -641,4 +643,19 @@ public static void setOptions(ServiceClient client, String accessUsername, Strin
 
         IdentityManagementServiceUtil.setAutheticationOptions(client, accessUsername, accessPassword);
     }
+
+    /**
+     * Build Tenant Qualified Url.
+     *
+     * @param context Context.
+     * @return Tenant Qualified Url.
+     */
+    public static String buildTenantQualifiedUrl(String context) {
+
+        try {
+            return ServiceURLBuilder.create().addPath(context).build().getAbsoluteInternalURL();
+        } catch (URLBuilderException e) {
+            throw new IdentityRuntimeException("Error while building url for context: " + context);
```

Comment:
Can't we throw the error and handle at the places where we consume the API?

Reply:
Updated the PR with the fix.